node/pegnet/bank.go

```diff
@@ -0,0 +1,126 @@
+package pegnet
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/pegnet/pegnet/modules/conversions"
+)
+
+// BankEntry is for managing the PEG Bank.
+// The bank is the amount of PEG allowed to be converted into every block.
+// This table should be used for bank management purposes.
+//
+// The bank has the following data structure at each height. It can be
+// used to control the amount of PEG allowed to be converted into at each
+// pegnet block.
+//
+// There is extra information provided for informational purposes. The only
+// true value needed here is likely the BankAmount and possible the BankUsed.
+// The PEGRequested shows the demand for PEG at a high level.
+type BankEntry struct {
+	// Each height will have a bank entry.
+	Height int32
+	// BankAmount is the total amount of PEG allowed to be converted into
+	// for the given height.
+	BankAmount int64 // units are PEGtoshi
+	// BankUsed is some additional information to detail how much of the bank
+	// was consumed.
+	BankUsed int64 // units are PEGtoshi
+	// PEGRequested is the total amount of PEG requested
+	PEGRequested int64 // units are PEGtoshi
+}
+
+const createTableBank = `CREATE TABLE IF NOT EXISTS "pn_bank" (
+	"height" INTEGER PRIMARY KEY,
+	"bank_amount" INTEGER NOT NULL DEFAULT 0,
+	"bank_used" INTEGER NOT NULL DEFAULT 0,
+	"total_requested" INTEGER NOT NULL DEFAULT 0,
+	
+	UNIQUE("height")
+);
+`
+
+const (
+	// Bank Properties and constants
+
+	// BankBaseAmount is the minimum the bank size will ever be.
+	// Any bank growth will be ontop of this value.
+	BankBaseAmount = conversions.PerBlock
+
+	// BankGrowth
+	BankGrowthAmount = 500 * 1e8
+	BankDecayAmount  = BankGrowthAmount
+
+	// BankMaxLimit is the maximum size of the Bank for any given block
+	// 288 blocks of growth
+	BankMaxLimit = BankBaseAmount + (288 * BankGrowthAmount)
```

Comment:
These Bank* variables are no longer needed for anything

Reply:
Removed the constants